### ISSUE TITLE
[med] Keyboard Navigation: Terms of service: No 'skip to main content' link implemented

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5504]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to the 'terms of service' link using Tab key and press Enter key.
3. On 'Terms of service' page, navigate through the page using Tab key.
4. Note if there is any skip link to navigate directly to the main content.

## Expected Result:
Right at the beginning of the page, there should be a link which allows to skip the navigation elements and continue directly with the main page content.

## Actual Result:
The link 'Skip to main content' is not provided. User has to tab through the elements in the top and left of the page to get to the main content of the page.

Other occurrences:
1. A similar issue is seen on Privacy policy and Licenses pages.
2. Also seen on Chat page.

## Workaround:
No workaround

## Area issue was found in:
Terms of service page, Licenses page, Privacy policy page, Chat page

## Failed WCAG checkpoints
2.4.1

## User impact:
The keyboard users will have to navigate through the elements in the top and left of page before accessing main page.

## Suggested resolution:
Implement a 'skip to main content' link which will allow users to bypass the navigation elements and proceed directly with the main content of the page. See: https://www.w3.org/WAI/WCAG21/Techniques/general/G1

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134722016-2539fca9-77e8-4596-8ba3-a1f998ce615a.mp4

![Bug5182245_Terms_of_services_-_bypass_blocks](https://user-images.githubusercontent.com/88733897/134722032-469198c0-8a0f-436b-9df8-6ffa956ed76d.png)
![Bug5182245_License_-_bypass_blocks](https://user-images.githubusercontent.com/88733897/134722034-c4e9d071-f8d5-4e01-aab0-fceff2aebd0f.png)
![Bug5182245_Privacy_policy_-_bypass_blocks](https://user-images.githubusercontent.com/88733897/134722041-c4ee8249-060a-41db-822f-ca7b953af2f3.png)
![Other-Chat-No_skip_links_or_other_mechanisms_to_jump_to_sections_of_page](https://user-images.githubusercontent.com/88733897/134722047-880ccfbb-6425-4c3e-b2d4-4a8f543460a9.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
Same comment: So the terms of service page is hosted on UseDot and it's technically not part of the NewDot app. @stitesExpensify @kadiealexander do you still think it's worth addressing these issues, or should we punt and focus on the app itself?
